Competition2017/src/competition/subsystems/drive/commands/DriveForDistanceCommand.java

```diff
@@ -120,6 +120,9 @@ public boolean isFinished() {
     
     @Override
     public void end() {
+        log.info("Ending, PreviousPosition was " + previousPositionInches 
+                + ", Targeted Delta in distance is " + deltaDistance + " Distance traveled is " 
```

Comment:
Yeah, as part of that, I'd recommend a period after "Ending" instead of a comma.